src/site/_includes/homepage.njk

```diff
@@ -87,10 +87,23 @@ eleventyComputed:
     >
       {{ i18n.about_cta }}
     </a>
+  </section> #}
+
+  <section class="w-promo-banner w-layout-container">
+    <img src="/images/modules/cds20-wide_960.jpg" alt="" width="480" height="270">
+    <p>
+      Chrome Dev Summit 2020 is back on December 9-10, but this time we're
+      coming to you. We’re bringing all the latest updates, just as much
+      content, and many more Chromies. Check out the
+      <a href="http://developer.chrome.com/devsummit/?utm_source=webdevhp&utm_medium=website">
+        <span class="w-visually-hidden">Chrome Dev Summit</span>website
```

Comment:
Can we put a space before "website"? Screen readers will probably abut these words.